### PR TITLE
Add IIT_V6 to support 6-element vectors in intrinsics.

### PR DESCRIPTION
**Pull request by rampitec**
Needed for the future patch.

### CODE REVIEW

File: llvm/include/llvm/IR/Intrinsics.td [hunk 1]
```diff
@@ -272,28 +272,29 @@ def IIT_F32  : IIT_VT<f32,  7>;
...
-def IIT_STRUCT4 : IIT_Base<23>;
-def IIT_STRUCT5 : IIT_Base<24>;
-def IIT_EXTEND_ARG : IIT_Base<25>;
-def IIT_TRUNC_ARG : IIT_Base<26>;
-def IIT_ANYPTR : IIT_Base<27>;
-def IIT_V1 : IIT_Vec<1, 28>;
-def IIT_VARARG : IIT_VT<isVoid, 29>;
-def IIT_HALF_VEC_ARG : IIT_Base<30>;
-def IIT_SAME_VEC_WIDTH_ARG : IIT_Base<31>;
+def IIT_V6   : IIT_Vec<6,  11>;
```
Comment (chapuni) [inline llvm/include/llvm/IR/Intrinsics.td:275]: You should avoid lower numbering unless it is used often. `<=16` is for short encoders.

**Review by chapuni — CHANGES_REQUESTED** [review]

File: llvm/include/llvm/IR/Intrinsics.td (hunk 1 above)
Comment (rampitec) [reply llvm/include/llvm/IR/Intrinsics.td]: Thanks! Moved to the end.

**Review by chapuni — APPROVED** [review]
LGTM. Please wait for one more approval.

**Comment by rampitec** [comment]
> LGTM. Please wait for one more approval.

Ack. Thanks for review!

**Review by arsenm — APPROVED** [review]

**Pull request merged**